modules/cardano-service.nix

```diff
@@ -45,7 +45,7 @@ let
     (optionalString (!cfg.enableP2P) "--kademlia-explicit-initial --disable-propagation ${smartGenPeer}")
     "--topology ${cfg.topologyYaml}"
     "--node-id ${params.name}"
-    "--config-name testnet_staging_full"
+    "--custom-config-name testnet_staging_full"
```

Comment:
I wonder, is this required to be set?

Reply:
Yes, this night's deployment failed because of:

1. Duncan needed a new topo file
2. The Cardano changes required to accomodate it were in `master`
3. `master` now takes this flag to choose the genesis it starts with -- precisely the direction we want, but in this case for me it meant..
4. Having to wait for the morning, until George tells me about the config changes.

Reply:
All of this is in the Slack archives of #devops and #sl..

Reply:
Because we've communally agreed we need to go back, and re-deploy the old staging, this needs to be reverted, of course (old Cardano won't accept this option).